### PR TITLE
Ins 26309 and Ins 26409

### PR DESCRIPTION
**Pull request by ruhirInstaclustr**

### CODE REVIEW

File: docs/data-sources/kafka_connect_cluster_v2_instance.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following terms are used to describe attributes in the schema of this data s
 <br>Status of the cluster.<br><br>
 *___kafka_connect_version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka connect to run on the cluster. Available versions: <ul> <li>`3.3.1`</li> <li>`3.5.1`</li> <li>`3.1.2`</li> <li>`3.4.1`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka connect to run on the cluster. Available versions: <ul> <li>`3.3.1`</li> <li>`3.5.1`</li> <li>`3.1.2`</li> <li>`2.8.2`</li> <li>`3.4.1`</li> <li>`3.0.2`</li> </ul><br><br>
```
Comment (anamarim-instaclustr) [inline docs/data-sources/kafka_connect_cluster_v2_instance.md:39]: Is 2.8.2 still an available Kafka Connect version?

File: docs/resources/kafka_connect_cluster_v2.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The following terms are used to describe attributes in the schema of this resour
 <ins>Constraints</ins>: allowed values: [ `PRODUCTION`, `NON_PRODUCTION` ]<br><br>SLA Tier of the cluster. Non-production clusters may receive lower priority support and reduced SLAs. Production tier is not available when using Developer class nodes. See [SLA Tier](https://www.instaclustr.com/support/documentation/useful-information/sla-tier/) for more information.<br><br>
 *___kafka_connect_version___*<br>
 <ins>Type</ins>: string, required, immutable<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka connect to run on the cluster. Available versions: <ul> <li>`3.3.1`</li> <li>`3.5.1`</li> <li>`3.1.2`</li> <li>`3.4.1`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka connect to run on the cluster. Available versions: <ul> <li>`3.3.1`</li> <li>`3.5.1`</li> <li>`3.1.2`</li> <li>`2.8.2`</li> <li>`3.4.1`</li> <li>`3.0.2`</li> </ul><br><br>
```
Comment (anamarim-instaclustr) [inline docs/resources/kafka_connect_cluster_v2.md:54]: Same here

**Review by icalastaid — DISMISSED** [review]
Thanks for the updates! Looks good to me.

File: docs/data-sources/kafka_cluster_v2_instance.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ Adds the specified version of Kafka REST Proxy to this Kafka cluster.<br>
 <br>Integrates the REST proxy with the Schema registry attached to this cluster. Requires 'integrateRestProxyWithSchemaRegistry' to be 'true'.<br><br>
 *___version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka REST Proxy to the Kafka cluster. Available versions: <ul> <li>`5.0.4`</li> <li>`5.0.0`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka REST Proxy to the Kafka cluster. Available versions: <ul> </ul><br><br>
```
Comment (iclonghoang) [inline docs/data-sources/kafka_cluster_v2_instance.md:122]: is this add-on moving off GA?
there's no version available

File: docs/data-sources/kafka_cluster_v2_instance.md [hunk 4]
```diff
@@ -341,11 +341,11 @@ Adds the specified version of Kafka Karapace Schema Registry to this Kafka clust
...
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka Schema Registry to the Kafka cluster. Available versions: <ul> <li>`3.4.3`</li> <li>`3.6.2`</li> </ul><br><br>
+<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka Schema Registry to the Kafka cluster. Available versions: <ul> <li>`3.6.2`</li> </ul><br><br>
 <a id="nested--schema_registry"></a>
 ## Nested schema for `schema_registry`
 Adds the specified version of Kafka Schema Registry to this Kafka cluster.<br>
 ### Read-only attributes
 *___version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Adds the specified version of Kafka Schema Registry to the Kafka cluster. Available versions: <ul> <li>`5.0.0`</li> </ul><br><br>
```
Comment (iclonghoang) [inline docs/data-sources/kafka_cluster_v2_instance.md:351]: same as above, empty list here

File: docs/resources/kafka_topic_v2.md [hunk 5]
```diff
@@ -10,6 +10,136 @@ Definition of a Kafka Topic to be applied to a Kafka cluster.
 ```
 resource "instaclustr_kafka_topic_v2" "example" {
   partitions = 3
+  configs {
```
Comment (iclonghoang) [inline docs/resources/kafka_topic_v2.md:13]: can you add `x-terraform-field-name: config` to this property in API spec?
properties of array type is better named as singular noun in TF. `configs` -> `config`
more details https://instaclustr.atlassian.net/wiki/spaces/IDS/pages/2241658905/Developing+for+API+v2+in+a+Terraform+Compliant+manner

File: docs/data-sources/kafka_cluster_v2_instance.md (hunk 3 above)
Comment (ruhirInstaclustr) [reply docs/data-sources/kafka_cluster_v2_instance.md:122]: yes true, no GA for this add-on from now. There are karapace addons replacing this.

File: docs/data-sources/kafka_cluster_v2_instance.md (hunk 4 above)
Comment (ruhirInstaclustr) [reply docs/data-sources/kafka_cluster_v2_instance.md:351]: yes.

**Review by iclonghoang — APPROVED** [review]

File: docs/resources/kafka_topic_v2.md (hunk 5 above)
Comment (iclonghoang) [reply docs/resources/kafka_topic_v2.md:13]: approved as api is deployed in preprod
and this is not wrong functionally, just minor TF schema inconsistency

**Review by iclonghoang — APPROVED** [review]

**Review by parislarkins — APPROVED** [review]

**Pull request merged**